DDCore/CMakeLists.txt

```diff
@@ -156,7 +154,7 @@ install(DIRECTORY
   )
 
 set_target_properties(DDCore DDCorePlugins PROPERTIES VERSION ${DD4hep_VERSION} SOVERSION ${DD4hep_SOVERSION})
-install(TARGETS DDCore DDCorePlugins ${OPTIONAL_TARGETS} EXPORT DD4hep
```

Comment:
We still want to install the plugins, but not export them as targets?
Or where is the installation being asked for?